onnx/test/test_backend_test.py

```diff
@@ -105,7 +105,16 @@ def do_enforce_test_coverage_safelist(model: ModelProto) -> bool:
     return all(node.op_type not in {"RNN", "LSTM", "GRU"} for node in model.graph.node)
 
 
-backend_test = onnx.backend.test.BackendTest(DummyBackend, __name__)
+test_kwargs = {
+    "test_affine_grid_2d_align_corners_expanded": {"strict_mode": False},
+    "test_affine_grid_2d_expanded": {"strict_mode": False},
+    "test_affine_grid_3d_align_corners_expanded": {"strict_mode": False},
+    "test_affine_grid_3d_expanded": {"strict_mode": False},
+}
```

Comment:
I would add a note on why we need to turn off strict mode, and if there are plans to fix, create an issue an link it here?